python/tvm/driver/tvmc/composite_target.py

```diff
@@ -23,7 +23,8 @@
 import tvm.contrib.target.vitis_ai  # pylint: disable=unused-import
 
 from tvm.relay.op.contrib.arm_compute_lib import partition_for_arm_compute_lib
-from tvm.relay.op.contrib.ethosn import partition_for_ethosn
+from tvm.relay.op.contrib.ethosn import partition_for_ethosn77
+from tvm.relay.op.contrib.ethosn import partition_for_ethosn78
```

Comment:
These can be combined into one import:
`from tvm.relay.op.contrib.ethosn import partition_for_ethosn77, partition_for_ethosn78`

Reply:
Fixed now.